widgets/accordion/accordion.php

```diff
@@ -193,9 +194,14 @@ public function get_template_variables( $instance, $args ) {
 					$id .= '-' . $args['widget_id'];
 				}
 				$panel['anchor'] = $id . '-' . $i;
+			} else if ( isset( $anchor_list[ strtolower( $panel['title'] ) ] ) ) {
```

Comment:
This doesn't work when you have titles with uppercase letters, because here it's checking using `strtolower`, but the anchors are being added to `$anchor_list` below as is. Perhaps when you make changes just do a quick test again to make sure things are working as expected.